client/src/app/views/form.coffee

```diff
@@ -0,0 +1,28 @@
+$ = require('jquery')
+Marionette = require('backbone.marionette')
+template = require('../templates/form.jade')
+errorMessage = require('../templates/error.jade')
+SiteModel = require('../models/site')
+
+module.exports = Marionette.ItemView.extend(
+  model: new SiteModel()
+  template: template
+  events:
+    'submit form' : 'enterSite'
+  ui:
+    url: '#url'
+    email: '#email'
+  enterSite: (e) ->
+    e.preventDefault()
+    if !this.validateForm() then return
```

Comment:
Since we now converted to coffeescript lets try to use the language features when able.

    return unless @validateForm()